crates/exex/test-utils/src/lib.rs

```diff
@@ -238,7 +238,7 @@ pub async fn test_exex_context_with_chain_spec(
     chain_spec: Arc<ChainSpec>,
 ) -> eyre::Result<(ExExContext<Adapter>, TestExExHandle)> {
     let transaction_pool = testing_pool();
-    let evm_config = EthEvmConfig::default();
+    let evm_config = EthEvmConfig::new(MAINNET.clone());
```

Comment:
same here, we already have a `chain_spec` arg

Reply:
done